### ISSUE TITLE
User unable to access News Feed

### ISSUE THREAD

**Issue by MelissaHanna** [tbs-sct/gcconnex#625]
Contact: sebnem.oren@phac-aspc.gc.ca 

User message below: 
I'm not able to access the News Feed. The error message is "The gcconnex.gc.ca page isn’t working gcconnex.gc.ca is currently unable to handle this request. HTTP ERROR 500" Is there a setting I need to change? Issue has been ongoing for quite a while now (several weeks at least).

**Comment by KevinChagnon** [comment]
Same as # 538

**Comment by ethanWallace** [comment]
@KevinChagnon - is the user still having this issue?

**Comment by KevinChagnon** [comment]
Yes, the user is still getting an error 500 when trying to access the Newfeed.  He can otherwise navigate through GCConnex without further issues.
